pkg/ccl/sqlproxyccl/tenant/directory_cache.go

```diff
@@ -462,12 +462,7 @@ func (d *directoryCache) updateTenantEntry(ctx context.Context, pod *Pod) {
 		} else {
 			log.Infof(ctx, "updated IP address %s with load %.3f for tenant %d", pod.Addr, pod.Load, pod.TenantID)
 		}
-	// Update entries of UNKNOWN pods only if they are already present.
-	case UNKNOWN:
-		if entry.UpdatePod(pod) {
-			log.Infof(ctx, "updated IP address %s with load %.3f for tenant %d", pod.Addr, pod.Load, pod.TenantID)
-		}
-	default:
+	case DELETING:
 		// Remove addresses of DELETING pods.
 		if entry.RemovePodByAddr(pod.Addr) {
```

Comment:
What do you think of having a default case that logs the unknown event?

Reply:
My only concern with that is that I worry we'll get a lot of spammy sqlproxy logs during the rollout phase (i.e. with the current tenant directory implementation). These UNKNOWN pod events are emitted whenever load changes, which should be very frequent.

Reply:
I'm happy to revisit this decision to log these unknown events once we remove the load watcher in the current tenant directory implementation. What do you think?

Reply:
Ideally we shouldn't be getting anymore UNKNOWN pod events, but I think logging would still be helpful in the long run.